### DISCUSSION TITLE
wodnjse2

### DISCUSSION THREAD

**Discussion by learningssik** [learningssik/pull-request#21] [Q&A]
comment here

**Comment by jaeyunha** [comment]
Thank you so much for your blog post #2 
I was glad to get the badges! haha

**Reply by learningssik** [reply]
That's cool! I am very happy that it was helpful to you.

**Reply by learningssik** [reply]
If this article was helpful, please click the `Star` in the upper right corner of this repository ^-^

**Comment by devwontory** [comment]


**Reply by learningssik** [reply]
#27 에 댓글을 추가하세요!

**Reply by learningssik** [reply]
이 글이 도움이 되셨다면 본 레파지토리 우측 상단의 Star를 눌러주세요 ^-^
